examples/with-rbx-bulma-pro/pages/about.js

```diff
@@ -0,0 +1,21 @@
+import {Section, Title} from 'rbx';
+import Layout from '../components/Layout';
+
+const AboutPage = () => (
+  <Layout>
+    <Section>
+      <Title as='h2'>About</Title>
+      <p>That's exactly what we do with Next.js. Instead of PHP, we build the app with JavaScript and React. Here are some other cool features Next.js brings to the table:</p>
```

Comment:
Why does it start like that?, makes no sense to me.

Reply:
I copied text from [getting start guide](https://nextjs.org/learn/basics/getting-started)